src/macro.jl

```diff
@@ -1165,9 +1165,9 @@ function make_many_actors(act!, args, ex1, outer::Vector, ex3, inner::Vector, ex
 
                     local @inline function $act!(::Type{<:CuArray}, $(args...), $KEEP=nothing, $FINAL=true) where {$TYP}
                         $info2
-                        cu_kern! = $kernel(CUDADevice(), $(store.cuda))
+                        cu_kern! = $kernel(CUDADevice())
                         $(asserts...)
-                        $ACC = cu_kern!($(args...), $KEEP, $FINAL; ndrange=tuple($(sizes...)), dependencies=Event(CUDADevice()))
+                        $ACC = cu_kern!($(args...), $KEEP, $FINAL; ndrange=tuple($(sizes...)), workgroupsize=$(store.cuda), dependencies=Event(CUDADevice()))
```

Comment:
... and that variable can be pasted in here:
```suggestion
                        $ACC = cu_kern!($(args...), $KEEP, $FINAL; ndrange=tuple($(sizes...)), workgroupsize=$workgroupsize, dependencies=Event(CUDADevice()))
```

Reply:
```suggestion
                        $ACC = cu_kern!($(args...), $KEEP, $FINAL; ndrange=tuple($(sizes...)), workgroupsize=workgroupsize, dependencies=Event(CUDADevice()))
```
(since `workgroupsize` is a variable inside the expression)